torch/_dynamo/compiled_autograd.py

```diff
@@ -401,11 +416,12 @@ def runtime_wrapper(compiled_fn, inputs, sizes, scalars, hooks):
                 for i in runtime_inputs_to_move:
                     inputs[i] = inputs[i].pin_memory().cuda(non_blocking=True)
 
-                with _disable():
+                with _disable(), make_compile_context(self.id):
```

Comment:
@ezyang this function is called at runtime, and we call into the torch.compile'd function from within a top-level compile context

Reply:
Err... but why do you need this? Is it similar reasoning to capturing runtime autotuning / cudagraphify discussion we had for regular forward-backward? But in that case, we specifically wrote the logic there so it was as low overhead as possible (only a single conditional in the exit case, likely to be False so never exercise code at all.)

I'm happy to let you make your bed for compiled autograd but this will probably eventually show up as runtime overhead!